pages/404.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+export default function NotFound() {
+  return (
+    <div className="flex flex-col justify-center items-center w-screen h-screen">
+      <h1 className="font-semibold text-2xl">404</h1>
+      <p className="mx-4 w-96 text-center text-xl">
+        Sorry, the page you wanted was not found. try serching for something
+        else.
+      </p>
+      <button className="btn-secondary">
```

Comment:
add some margins to the button
![image](https://user-images.githubusercontent.com/12988551/136421521-7cebf9de-5385-484a-a214-f42f981d7db2.png)